vulkan_generator/vulkan_parser/parser.py

```diff
@@ -50,13 +51,16 @@ def parse(filename: Path) -> types.VulkanMetadata:
     tree = ET.parse(filename)
     all_types = types.AllVulkanTypes()
     all_commands = types.AllVulkanCommands()
+    spirv_metadata = types.SpirvMetadata()
 
-    for child in tree.iter():
+    for child in tree.getroot():
```

Comment:
Possibly should have put this in the other PR but it's a minor nit. Don't change anything here.